specs/angular-readonly-form-styling/README.md

```diff
@@ -0,0 +1,152 @@
+# Angular read-only form styling
+
+## Problem Statement
+
+There is a desire to render Angular forms with read-only controls rather than disabled controls when the control in the Angular form is marked as disabled. This is because there are use cases where the configuration represented by nimble controls is not editable but is still relevant and should be easily readable. Additionally, the only way to mark something in an Angular form as "not editable" is to mark it as disabled (i.e. there is no concept of "read only" in an Angular form). However, there is quite a bit of logic built into Angular forms and Angular's control value accessors around what it means to render a component bound to a disabled form control.
+
+With some changes to nimble's controls and nimble's Angular control value accessors, then Angular applications could much more easily create forms with read-only controls rather than disabled controls.
+
+An example can be seen below of a page that currently has disabled controls and the ideal UI.
+
+Current UI with disabled controls:
+![disabled form](disabled-form.png)
```

Comment:
```suggestion

![disabled form](disabled-form.png)
```
In my browser, it's putting the image directly to the right of the text. Perhaps a newline fixes that?